src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -748,6 +748,18 @@ def __call__(self):
         cmsswStep = self.step.getTypeHelper()
         for om in cmsswStep.listOutputModules():
             mod = cmsswStep.getOutputModule(om)
+            modName = str(getattr(mod, "_internal_name"))
```

Comment:
Kenyi, how about defining a new "public" method in the ConfigSection object (src/python/WMCore/Configuration.py), to retrieve the object name? Something like:
```
def getInternalName(self):
    return self._internal_name
```

The default value is None, so whoever is getting it, needs to take that into consideration.

This `str()` casting here might become a problem as well, if the config section name is a byte string, it would result in `'b"name"'`.

Reply:
Good point. I called it that way because it's how it was used in WMTweak:
https://github.com/dmwm/WMCore/blob/psettweak/src/python/PSetTweaks/WMTweak.py#L519

But having a public method and modify both would make it more python3 friendly, thinking about the str & bytes thing.